docs/source/deploying/k8s.md

```diff
@@ -0,0 +1,178 @@
+# Using Kubernetes
+
+The vLLM Documentation on [Deploying with Kubernetes](https://docs.vllm.ai/en/latest/deployment/k8s.html) is a comprehensive guide for configuring deployments of models on kubernetes. This guide highlights some key differences when deploying on kubernetes with Spyre accelerators.
+
+## Deploying on Spyre Accelerators
+
+**Pre-requisite** Ensure that you have a running Kubernetes cluster with Spyre accelerators.
```

Comment:
Nit: isn't `prerequisite` one word?

Reply:
Also worth having it as an Admonition?

Reply:
👍 

I had just copied from vllm docs, but will change